Analysis/Tasks/multiplicityTable.cxx

```diff
@@ -46,9 +46,14 @@ struct MultiplicityTableTask {
     float multZNA = zdc.energyCommonZNA();
     float multZNC = zdc.energyCommonZNC();
 
-    LOGF(debug, "multV0A=%5.0f multV0C=%5.0f multZNA=%6.0f multZNC=%6.0f", multV0A, multV0C, multZNA, multZNC);
+    int multTracklets = 0;
+    for (auto& tr : tracks)
```

Comment:
The faster way would be to do a partition and get its size instead of looping over the whole table. Or even add a filter and get directly the size of the table that you get as an argument to process().